src/pages/AboutUser/AboutUser.jsx

```diff
@@ -62,6 +69,7 @@ const AboutUser = () => {
           </div>
         </div>
       </div>
+      <LeafletMap  cordinates={arrofCord.length ? defaultCord : arrofCord} name={name}/>
```

Comment:
does it works? 
Because in my comment condition was another.
Here we have if arrofCord.length then we will use defaultCord